3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -422,17 +422,23 @@ I membri dell'organizzazione sono prestabiliti. Qualora le competenze interne al
 
 Non sono previste variazioni della composizione dell'organizzazione, se non in via straordinaria e comunque discussa preventivamente con il Committente.
 
-=== Processo di gestione della qualità <processo_gestione_qualità>
+=== Processo di Gestione della Qualità <processo_gestione_qualità>
 
 _Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.2.5_
 
 ==== Scopo
+Il processo di Gestione della Qualità ha lo scopo di garantire che i prodotti del progetto soddisfino gli obiettivi di qualità del gruppo e i bisogni del Proponente.
 
-Il processo di gestione della qualità ha lo scopo di garantire che i prodotti del progetto soddisfino gli obiettivi di qualità del gruppo e i bisogni del Proponente.
-
-=== Attività
-==== Pianificazione
+==== Risultati
+Come risultato dell'efficace attuazione del processo di Gestione della Qualità:
+- vengono definiti e implementati gli obiettivi, le politiche e le procedure di Gestione della Qualità del gruppo;
+- vengono stabiliti i criteri e i metodi di valutazione della qualità;
+- vengono fornite risorse e informazioni ai progetti per supportare l'operazione e il monitoraggio delle attività di garanzia della qualità del progetto;
+- vengono raccolti e analizzati i risultati delle valutazioni della garanzia della qualità;
+- le politiche e le procedure di Gestione della Qualità vengono migliorate basandosi sui risultati dei progetti e dell'organizzazione.
 
+==== Attività
+===== Pianificazione
 Il gruppo #err418 pone i seguenti principi di qualità:
 
 + tutto il materiale presente nei branch principali della repository (`src` e di conseguenza `main`) deve essere di buona qualità:
```

Comment:
Modificare il contenuto delle parentesi in "(`src` e, di conseguenza, `main` per Documenti, e `dev` per WMS3)"